test/navigation/NavigatorSpec.scala

```diff
@@ -3010,6 +3013,18 @@ class NavigatorSpec extends SpecBase with BeforeAndAfterEach {
           ) mustBe routes.DownloadRequestSuccessController.onPageLoad()
         }
       }
+
+      "in New UKIMS Number update Journey" - {
+        "must go from UkimsNumberUpdatePage to CyaMaintainProfilePage" in {
```

Comment:
typo in test

Reply:
Here, we need to call `submitTraderProfile` to set `eoriChange` to `false`.

Reply:
thank you  :)